dev/doc/debugging.md

```diff
@@ -59,6 +59,25 @@ Debugging from Caml debugger
      from the debugger. If this happens, unset the variable, re-start Emacs, and 
      run the debugger again.
 
+Debugging with ocamldebug from the command line
+===============================================
+
+1. (for Cygwin/Windows) Pass the -no-custom option to the configure script before building Coq.
+2. Run "make byte"
+3. (for Cygwin/Windows) Add the full pathname of the directory .../kernel/byterun to your bash PATH.
+   Alternatively, copy the file kernel/byterun/dllcoqrun.dll to a directory that is in the PATH.  (The
+   CAML_LD_LIBRARY_PATH mechanism described at the end of INSTALL isn't working.)
+4. Run "dev/ocamldebug-coq bin/coqtop.byte.exe"
```

Comment:
Maybe, this line should not contain the `.exe` extension since it is Windows-only (you can maybe use a parenthesis to remind about this extension for Cygwin/Windows).